contracts/src/v0.8/functions/dev/1_0_0/FunctionsRouter.sol

```diff
@@ -224,9 +224,7 @@ contract FunctionsRouter is RouterBase, IFunctionsRouter, FunctionsSubscriptions
     uint96 juelsPerGas,
     uint96 costWithoutFulfillment,
     address transmitter
-  ) external override whenNotPaused returns (uint8 resultCode, uint96 callbackGasCostJuels) {
```

Comment:
why is whenNotPaused removed here?

Reply:
Oops, that one should still be there as a internal function call.
Good eyes.